### PR TITLE
Unexport var git.GlobalCommandArgs

### PR DESCRIPTION
**Pull request by 6543**
as title

### CODE REVIEW

File: modules/git/command.go [hunk 1]
```diff
@@ -278,3 +278,19 @@ func (c *Command) RunTimeout(timeout time.Duration) (string, error) {
 func (c *Command) Run() (string, error) {
 	return c.RunTimeout(-1)
 }
+
+// HelperForTestsToAllowLFSFilters should only be used for tests
+func HelperForTestsToAllowLFSFilters() []string {
```
Comment (Gusted) [inline modules/git/command.go]:
```go
func ThatsOneBigVeryDescriptiveNameForASimpleFunctionThatActAsAHelperForTests()
```

**Review by Gusted — APPROVED** [review]

File: modules/git/command.go (hunk 1 above)
Comment (6543) [reply modules/git/command.go]: well I could not add it to command_test.go ... and it should not be used outside of tests - that should make it clear :D

Comment (zeripath) [reply modules/git/command.go]: I think we should just call this `AllowLFSFiltersArgs()` 

By all means stick in the comment that it is intended only to be used in tests but this name is horrendous.

Comment (6543) [reply modules/git/command.go]: done

**Review by Gusted — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Pull request merged**